### ISSUE TITLE
GameJolt Client Game download always pending/queuing?

### ISSUE THREAD

**Issue by jianmingyong** [gamejolt/issue-tracker#545]
When downloading a game, let say I want to play Pokemon3D, I click on the download button and wait.

After a while, it will stuck at 90++% and when you browse other game and come back, it will be queuing up forever with 0%.

I have looked at the download folder of the gamejolt client and look at the zip file it generates. It is incomplete and doesn't match the exact checksum from the zip file on GameJolt website.

It also cannot stop or remove the game from gamejolt client. You have to manually delete the corrupted file that is being downloaded from gamejolt client.

I am sure I am not the only one, we have tons of player playing Pokemon3D and they face this issue as well. Manually download from website works. Not on gamejolt client. Please look into the download issue.

Using GameJolt Client on Windows 8.1.

**Comment by YLivay** [comment]
Is this for every game? Does the progress bar change from a solid one to striped one?

**Comment by jianmingyong** [comment]
I have tried again and here are the results:
http://i.imgur.com/MQu2XNp.png
http://i.imgur.com/XGkBjJZ.png
http://i.imgur.com/vsn9ubL.png

All games I tried doesn't seem to work. It is not working.

**Comment by YLivay** [comment]
Well, surprisingly you are the first with this specific issu haha.

It seems like the extraction phase itself doesnt even stwrt properly. In the next few days well release a version thatll allow us to debug ot further. Im guessing it has to do with lzma

**Comment by YLivay** [comment]
Yup, this does look like lzma and thanks to @addaleax adding pre-gyped versions looks like its starting to show signs of life! Will update here.

**Comment by addaleax** [comment]
@YLivay I just tried downloading a few things with the client – I guess striped progress bars correspond to extraction using lzma? Everything seems to work for me locally, but if I can help somewhere, please point me to it :)

**Comment by YLivay** [comment]
Thanks man! I think I got it working when we do a full package build on windows too :)
node-pre-gyp is magic.

**Comment by YLivay** [comment]
Should work in new version.
Wait till release!

**Comment by federicoebarros** [comment]
Hey sorry for bringing this topic back to life but I coudn't solve it by myself so far. I'm trying to download "Epistory - Typing Chronicles" but it's been all the time at 0% and I can't pause or cancel the download from the client that I installed from the AUR (yaourt command in x64 Manjaro Linux: aur/game-jolt-client 0.3.2-1). Every other game downloaded correctly but this remained after reinstalling the client, restarting the PC and logging out and in with my user in the client. I launched the client from the terminal and tried to pause and cancel de download so yoou can read the detail in the attached log file.

Distro information:
[fbarros@LenovoT460 game-jolt-client]$ uname -a
Linux LenovoT460 4.4.24-1-MANJARO #1 SMP PREEMPT Sat Oct 8 07:49:12 UTC 2016 x86_64 GNU/Linux

Any hints? If i completely remove the app, will it automatically reinstall all the games I downloaded today after installing the client and logging in or should I make a list of them? (from Game Jolt website my profile says I don't own any games)

Thanks in advance!

[gamejolt-console-log.txt](https://github.com/gamejolt/issue-tracker/files/518583/gamejolt-console-log.txt)
